### PR TITLE
Walk refactored as a component and added to the site

### PR DESCRIPTION
**Pull request by bjedrzejewski**
Walk is a component and is available on the site with some example.

### CODE REVIEW

**Comment by bjedrzejewski** [comment]
This closes #545

**Comment by bjedrzejewski** [comment]
@ColinEberhardt @chrisprice we think that this is ready to merge

File: site/src/components/data/walk.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  // data is generated here
+  var data = dataGenerator(100); // The series starts at 100
+
+  d3.select("#walk")
+    .text(JSON.stringify(data, null, 2));
+---
+
+The random walk component creates a random series of values based on a [Geometric Brownian Motion](http://stuartreid.co.za/interactive-stochastic-processes/). It 
+is useful for generating arbitrary sets of data. The [random financial](./financial.html) component uses it to generate two series of points, which are then converted into a price bar series.
```
Comment (ColinEberhardt) [inline site/src/components/data/walk.md]: > The random financial component uses it to generate two series of points, which are then converted into a price bar series.

This doesn't sound right to me - the mention of a bar series implies that this is a visual component that performs some rendering.

**Comment by ColinEberhardt** [comment]
> we think that this is ready to merge

This is looking good - although there is one minor comment that I'd like to see addressed. Thanks.

**Comment by ColinEberhardt** [comment]
Thanks @bjedrzejewski  & @jdunkerley

**Pull request merged**